### PR TITLE
Mobilsdk 1608

### PR DESCRIPTION
**Pull request by wayneburkett**

### CODE REVIEW

File: AdvancedSample/AdvancedSample/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                                <state key="normal" title="Check Eligibility"/>
+                                <connections>
+                                    <action selector="checkEligibilityButtonTouchUpInside:" destination="BYZ-38-t0r" eventType="touchUpInside" id="wdt-4Y-mCo"/>
+                                </connections>
+                            </button>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="wZA-DH-GmN" userLabel="Feedback Eligibility">
+                                <rect key="frame" x="20" y="337" width="374" height="30"/>
+                                <color key="backgroundColor" red="0.92156862750000001" green="0.16862745100000001" blue="0.23921568630000001" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <color key="tintColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                                <state key="normal" title="Check Feedback Eligibility"/>
```
[the remark this answers is not in the document]
Comment (wayneburkett) [reply AdvancedSample/AdvancedSample/Base.lproj/Main.storyboard]: Good point. I'll change this one.

File: AdvancedSample/AdvancedSample/Base.lproj/foresee_configuration.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "clientId": "Ij6P1lfZHchO/co10lQ4BQ==",
+  "notificationType": "IN_SESSION",
+  "measures":
+  [
+    {
+      "surveyId": "mobile_app-en",
+      "significantEventThresholds":
+      {
+        "app_test_1": 4
+      }
```
[the remark this answers is not in the document]
Comment (wayneburkett) [reply AdvancedSample/AdvancedSample/Base.lproj/foresee_configuration.json]: @awangvan Maybe as combined criteria? What do you think?

[the remark this answers is not in the document]
Comment (wayneburkett) [reply AdvancedSample/AdvancedSample/Base.lproj/foresee_configuration.json]: @awangvan adding a new criteria to the config is easy, but I'm not sure how to show it in the app. Any ideas?

[the remark this answers is not in the document]
Comment (awangvan) [reply AdvancedSample/AdvancedSample/Base.lproj/foresee_configuration.json]: Yeah combined criteria is a great idea. In terms of how to show it I think we need to guide the user to trigger it in the app. I.e., for pageViews criteria we could guide user to navigate through some demo pages.

File: AdvancedSample/AdvancedSample/Base.lproj/Main.storyboard (hunk 1 above)
[the remark this answers is not in the document]
Comment (wayneburkett) [reply AdvancedSample/AdvancedSample/Base.lproj/Main.storyboard]: ✅Updated

File: AdvancedSample/AdvancedSample/Base.lproj/foresee_configuration.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (wayneburkett) [reply AdvancedSample/AdvancedSample/Base.lproj/foresee_configuration.json]: ✅ I added a “page views” criteria and a HOWTO document that explains how to trigger the invite.

**Pull request merged**